test/e2e_node/node_shutdown_linux_test.go

```diff
@@ -57,6 +61,25 @@ var _ = SIGDescribe("GracefulNodeShutdown [Serial] [NodeFeature:GracefulNodeShut
 	f := framework.NewDefaultFramework("graceful-node-shutdown")
 	f.NamespacePodSecurityLevel = admissionapi.LevelPrivileged
 
+	ginkgo.BeforeEach(func() {
+		systemdVersion := 0
+		if _, err := exec.LookPath("systemd-run"); err == nil {
```

Comment:
if `systemd-run` isn't on the PATH (i.e. `err` is not `nil`), then we'll end up skipping the test due to the default value of `systemdVersion`. is that what we want?

Reply:
@cartermckinnon i can add additional logging for anyone looking at why things were skipped